themes/bootstrap3/js/facets.js

```diff
@@ -313,6 +318,7 @@ VuFind.register('multiFacetsSelection', function multiFacetsSelection() {
         toggleSelectedFacetStyle(elem);
       }
     }
+    saveUserSelectionLastState(isMultiFacetsSelectionActivated);
```

Comment:
While it doesn't hurt here, I think the proper place for this is inside the `if (typeof enable !== 'undefined') {` block above.